config/api.yml

```diff
@@ -1597,3 +1597,30 @@
       :delete:
       - :name: delete
         :identifier: zone_delete
+  :virtual_templates:
+    :description: Virtual Templates
+    :identifier: virtual_template
```

Comment:
@jntullo I'm just looking at the product feature you created and only saw `miq_virtual_template`, but I could be wrong?

The `Miq` prefix I believe came from having top level constant names clashing with those from third party libraries. I also think we're trying to avoid doing that now :wink: